components/AddForm.js

```diff
@@ -70,7 +73,7 @@ const StyledFormContainer = styled.form`
 const StyledInputField = styled.input`
   ${({ theme }) => `background-color:  ${theme.colors.fifth};
```

Comment:
```suggestion
  ${({ theme }) => css`
      background-color:  ${theme.colors.fifth};
      box-shadow: ${theme.boxShadow.shadow};
  `}
```
if you use the `css` helper function your template literal gets displayed as css.

Reply:
You need to import it from the styled-components before.